tests/activity_test.go

```diff
@@ -356,11 +359,13 @@ func (s *functionalSuite) TestActivityRetry() {
 		}
 	}
 
+	// let poller[1] to process tasks and assert that it completed successfully
```

Comment:
same as above, I think it would be most helpful to comment above the function definition. Right now the function definition comment reads `// PollAndProcessActivityTask for activity tasks` which is kinda redundant